component/ci-light/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM rust:1-alpine
+
+ARG BUILDEVENTS_VERSION=v0.17.0
+RUN apk add curl
```

Comment:
We could consider adding a couple Git-related packages as well, namely `github-cli openssh-client` as we tend to install these most of the time in the `alpine` images. However, we can also do this in followup work